src/app/src/pages/ManageRequests.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react"
+import { Box, List, ButtonGroup, Button , Badge, Typography, Divider} from "@mui/material"
+import NavBar from "../components/NavBar"
+import AdminAuthorCard from "../components/AdminAuthorCard"
+import AdminPostCard from "../components/AdminPostCard"
+import AdminNodeCard from "../components/AdminNodeCard"
+import ManageFollowRequests from "../components/ManageRequestsRequests"
+
+export default function ManageRequests(): JSX.Element {
+    const [listDisplay, setListDisplay] = React.useState({title:'Follow Requests',id:0});
+
+    //Some fake data to help with layouts
+    const followRequests = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        },
+    ];
+
+    const authors = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        profileImage:null,
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        profileImage:"",
+        },
+    ];
+
+    const posts=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        author:authors[0],
+        date: "2022-02-25"
+        }
+    ];
+
+    const nodes=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        username:"NodeOne",
+        }
+    ];
+
+    // Get length for badges
+    const totalRequests = followRequests.length;
+    const totalAuthors = authors.length;
+    const totalPosts = posts.length;
+    const totalNodes = nodes.length;
+
+    // Sidebar Button group
+    const buttons = [
+        <Button onClick={()=>setListDisplay({title:'Follow Requests',id:0})}key="requests" sx={{justifyContent:"space-between", display: "flex"}}> Requests <Badge badgeContent={totalRequests} color="secondary" sx={{justifyContent:"right", mx:3}}/></Button>,
```

Comment:
Too cluttered, abstract this possibly. Including inline styling within props for more than a couple of properties is bad, could lose marks.

Reply:
can you give me an example for abstracting this because idk where to begin

Reply:
```
Probably something like this.
let sx = {
justifyContent:"space-between",
display: "flex",
}

sx={sx}
```

Reply:
done

Reply:
Just realized that this is a part of a list. This can be generalized entirely. Ideally, your list would look something like this: 
```
const buttons = ["Requests",...,"Followers"]
```
and map this on line 112.

Reply:
done